src/OpenApi/test/Microsoft.AspNetCore.OpenApi.SourceGenerators.Tests/OperationTests.MinimalApis.cs

```diff
@@ -285,9 +291,11 @@ await SnapshotTestHelper.VerifyOpenApi(compilation, document =>
 
             var path14 = document.Paths["/14"].Operations[HttpMethod.Get];
             Assert.Equal("A summary of Get14.", path14.Summary);
+            Assert.Equal("Returns the greeting.", path14.Responses["200"].Description);
```

Comment:
Is there a way `<returns>` would apply to non-200 responses? Should we add a smoke test for that?

Reply:
Yes, this would happen if an endpoint returned a single `TypedResults.X` that wasn't related to a 200 OK status code. I can update the tests to reflect this.